cypress/integration/RichTextParser/URL_Detection.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Automatic url detection
+
+  Scenario Outline: Automatic URL detection
+      Given there is a text string that contains the URL of <URL>
+      And the URL is not preceded or followed immediately by a non-space character
+      Then the rendered URL is a <result>
+
+      Examples:
+        | URL                                        | result                                                                                              |
+        | duck.com                                   | <a href="http://duck.com">duck.com</a>                                                              |
```

Comment:
Complete tangent question - should we render these with `target="_blank"` with [`rel="noopener noreferrer"`](https://mathiasbynens.github.io/rel-noopener/)?  @cooper-joe

Reply:
Good question @amcgee, thanks for highlighting this. I think yes, those should be added by default. We can't know what user-generated content will be added here, so `_blank` is wise. Also definitely `rel="noopener noreferrer"`! I've added these to the scenario examples.